client/src/components/CheckoutForm/CheckoutForm.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import { render } from '@testing-library/react';
+import CheckoutForm from './CheckoutForm';
+
+jest.mock('@mui/material/Stepper', () => ({ children }) => <div>{children}</div>);
+jest.mock('@mui/material/Step', () => ({ children }) => <ul>{children}</ul>);
+jest.mock('@mui/material/StepLabel', () => ({ children }) => <li>{children}</li>);
+jest.mock('@mui/material/Button', () => ({ children }) => <button>{children}</button>);
+jest.mock('../../components/CheckoutSummary/CheckoutSummary', () => () => <div>CHECKOUT SUMMARY</div>);
+jest.mock('../../components/PaymentSuccess/PaymentSuccess', () => () => <div>PAYMENT SUCCESS</div>);
+jest.mock('../../components/UserDetailsForm/UserDetailsForm', () => () => <div>USER DETAILS FORM</div>);
+
```

Comment:
mocked `ShippingAddressForm` and `PaymentForm` are missing

Reply:
they werent added on purpose since in this snapshot they dont render at all, activeStep is alwats 0 which in return renders only the userDetails form.